webhook/main.go

```diff
@@ -0,0 +1,137 @@
+package main
```

Comment:
This should have a license header on it

`go run github.com/google/addlicense@v1.1.1 -c "The Authors (see AUTHORS file)" -l apache -v ./**/*.go`